server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileRef.java

```diff
@@ -36,7 +38,10 @@
  *   <li>by its key</li>
  *   <li>by the couple {language, name}</li>
  * </ul>
+ *
+ * @deprecated replaced by {@link QProfileReference}
  */
+@Deprecated
 public class QProfileRef {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)